src/wasm-lib/kcl/src/std/args.rs

```diff
@@ -874,21 +874,6 @@ impl<'a> FromKclValue<'a> for crate::std::polar::PolarCoordsData {
     }
 }
 
-impl<'a> FromKclValue<'a> for crate::std::planes::StandardPlane {
-    fn from_kcl_val(arg: &'a KclValue) -> Option<Self> {
-        let s = arg.as_str()?;
-        match s {
-            "XY" | "xy" => Some(Self::XY),
```

Comment:
If all these are deleted, how does `offsetPlane('XY', offset = 5)` still work?

Reply:
By the multiplication line

Reply:
Ah I see, PlaneData still has a FromKclValue impl that matches on these strings.